onnxruntime/core/providers/cuda/reduction/reduction_ops.cc

```diff
@@ -284,16 +285,34 @@ Status PrepareForReduce(const Tensor* X,
   std::vector<bool> reduced(rank, false);
   prepare_reduce_metadata.output_dims.reserve(input_dims.size());
   if (axes.size() > 0) {
+    int64_t reduced_axis;
+    std::vector<uint64_t> axes_(axes.size());
```

Comment:
if this is just a function-local, we can probably drop the " _ " in the name.

Reply:
keeping the names separate would make the code more readable

Reply:
Maybe I wasn't very clear, trailing underscores are for private class member variables. If function-local, we should do snake-case